lms/templates/oauth2_provider/authorize.html

```diff
@@ -0,0 +1,54 @@
+{% extends "main_django.html" %}
+{% with online_help_token="oauth2_provider_authorization" %}
+{% load theme_pipeline %}{% load sekizai_tags i18n configuration %}{% load url from future %}{% load staticfiles %}
```

Comment:
Actually...none of these seem to be used besides `i18n`. The unused libraries should be removed.